src/contexts/MenuContext.tsx

```diff
@@ -1,22 +1,42 @@
-import React from 'react';
+import React, { useState, useContext, createContext } from 'react';
 
-export interface State {
+interface MenuState {
```

Comment:
Everything looks good in this PR  👏, but I have one suggestion - we have many types / interfaces declared in this file `MenuContext.tsx` and other contexts inside  `/context` folder. At the beginning I didn't raise this problem, but we can leverage this PR and move all types to separate `types.ts` file. 

My motivation:

- gives us clear separation between context implementation and declared types
- makes contexts files much cleaner and more readable, by removing types declarations
- keeps consistent  with the rest types declared inside `types.ts`